dklearn/cross_validation.py

```diff
@@ -398,5 +412,5 @@ def train_test_split(*arrays, **options):
     if options:
         raise ValueError("Invalid parameters passed: %s" % str(options))
 
-    seed = different_seeds(1, random_state=random_state)[0]
+    seed, = random_state_data(1, random_state=random_state)
```

Comment:
Going off of https://twitter.com/raymondh/status/773786851727380480, which I found strange at first but it's grown on me :) Changed back to the `[0]` version.